### PR TITLE
add meta description

### PR DESCRIPTION
**Pull request by sumitshinde-84**
## Related Issue(s)
https://github.com/FlowFuse/website/issues/2177

## Checklist

 - [ ] I have read the [contribution guidelines](https://github.com/FlowFuse/flowfuse/blob/main/CONTRIBUTING.md)
 - [ ] I have considered the performance impact of these changes
 - [ ] Suitable unit/system level tests have been added and they pass
 - [ ] Documentation has been updated

### CODE REVIEW

File: src/ask-me-anything/ama-nodered-april.md [hunk 1]
```diff
@@ -1,6 +1,7 @@
 ---
 title: Ask Me Anything about Node-RED (April Edition)
 subtitle: Join Nick O'Leary, co-creator of Node-RED & CTO at FlowFuse and Rob Marcer, Developer Educator at FlowFuse, for an AMA on Node-RED
+description: Join Nick O'Leary and Rob Marcer for an Ask Me Anything on Node-RED! You ask, we answer. Get expert solutions now!
```
Comment (Hasmin-AC) [inline src/ask-me-anything/ama-nodered-april.md:4]:
```suggestion
description: Nick O'Leary and Rob Marcer answer your questions on Node-RED
```

File: src/ask-me-anything/ama-nodered-may.md [hunk 2]
```diff
@@ -1,6 +1,7 @@
 ---
 title: Ask Me Anything about Debugging Node-RED (May Edition)
 subtitle: Join Nick O'Leary, co-creator of Node-RED & CTO at FlowFuse and Rob Marcer, Developer Educator at FlowFuse, for an AMA on Node-RED Debugging
+description: Join Nick O'Leary and Rob Marcer for an Ask Me Anything on debugging Node-RED! You ask, we answer. Learn expert tips and tricks for Node-RED debugging. Register now!
```
Comment (Hasmin-AC) [inline src/ask-me-anything/ama-nodered-may.md:4]:
```suggestion
description: Nick O'Leary and Rob Marcer answer questions on debugging Node-RED in this Q+A session
```

File: src/ask-me-anything/ama-nodered.md [hunk 3]
```diff
@@ -1,6 +1,7 @@
 ---
 title: Ask Me Anything about Node-RED (March Edition)
 subtitle: Join Nick O'Leary, co-creator of Node-RED & CTO at FlowFuse and Rob Marcer, Developer Educator at FlowFuse, for an AMA on Node-RED
+description: Join Node-RED co-creator Nick O'Leary and Rob Marcer for an insightful AMA session. Ask your Node-RED questions directly and get expert advice and solutions. Don't miss out—register now!
```
Comment (Hasmin-AC) [inline src/ask-me-anything/ama-nodered.md:4]:
```suggestion
description: Node-RED co-creator Nick O'Leary and Rob Marcer answer your questions in this insightful AMA session on Node-RED
```

File: src/changelog/2023/12/email-alerting-node-red-crash.md [hunk 4]
```diff
@@ -1,5 +1,5 @@
 ---
-title: Email Alerts for Audit Log Events
+title: "FlowFuse update: Now receive email alerts for Audit Log events, ensuring timely notifications for Node-RED instance crashes."
```
Comment (Hasmin-AC) [inline src/changelog/2023/12/email-alerting-node-red-crash.md:2]: I think this should be description and not title?

Comment (sumitshinde-84) [reply src/changelog/2023/12/email-alerting-node-red-crash.md]: Oh, sorry. I've corrected it now.

File: src/about.njk [hunk 5]
```diff
@@ -26,10 +27,6 @@ meta:
         - https://twitter.com/flowfuseinc
         - https://github.com/FlowFuse
         - https://www.youtube.com/channel/UCbBzP8NZbv3WDtlt4UouA-g
-eleventyComputed:
```
Comment (ZJvandeWeg) [inline src/about.njk:29]: Why is this removed?

File: src/changelog/2023/10/certified-nodes.md [hunk 6]
```diff
@@ -1,5 +1,6 @@
 ---
 title: Certified Nodes
+description: "FlowFuse now offers certified nodes:  tested with proactive security measures to ensure system integrity and reliability, enhancing workflow efficiency and security."
```
Comment (ZJvandeWeg) [inline src/changelog/2023/10/certified-nodes.md:3]:
```suggestion
description: "FlowFuse now offers certified nodes: tested with proactive security measures to ensure system integrity and reliability, enhancing workflow efficiency and security."
```

**Review by ZJvandeWeg — CHANGES_REQUESTED** [review]

**Comment by ZJvandeWeg** [comment]
@sumitshinde-84 @Hasmin-AC What's the status of this PR?

File: src/about.njk (hunk 5 above)
Comment (sumitshinde-84) [reply src/about.njk]: I reverted it, what's the difference between ? 
```
meta:
        description:......
```

and 
````
meta:
        organization:
                            description:......
````

**Review by Hasmin-AC — APPROVED** [review]

**Review by ZJvandeWeg — APPROVED** [review]
